src/widgets/auto-style/style-utils.js

```diff
@@ -1,21 +1,26 @@
 var _ = require('underscore');
 var postcss = require('postcss');
+var stripInlineComments = require('postcss-strip-inline-comments');
+var SCSSsyntax = require('postcss-scss');
```

Comment:
You are right, looks like the last version is not working properly with our current jasmine thing. I'm using the same one from Grainstore (https://github.com/CartoDB/grainstore/blob/master/package.json#L26) and seems like it is alive again... let's see.